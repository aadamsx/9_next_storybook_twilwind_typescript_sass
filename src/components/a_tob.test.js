const rewire = require("rewire");
const a_tob = rewire("./a_tob");
const atob = a_tob.__get__("atob");
// @ponicode
describe("atob", () => {
  test("0", () => {
    let result = atob("Zm9vYmFy");
    expect(result).toMatchSnapshot();
    expect(result).toBe("foobar");
  });
});
