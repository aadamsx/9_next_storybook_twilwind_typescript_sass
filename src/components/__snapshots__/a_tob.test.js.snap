// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`atob 0 1`] = `"foobar"`;
